sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplate.java

```diff
@@ -120,13 +125,19 @@ public List<String> getBdaNames(){
         return List.of(daName.getStructNames().toArray(new String[0]));
     }
 
-    public <T extends DataTypeName> void addStructName(String structName, Class<T> cls){
```

Comment:
Why split in two methods while this method did the same thing?

Reply:
Genericity is not some time readable but allow to ovoid repetition for objects which methods do the same thing. But anyway your explanation is clear and . 
So it's good for me.